client/src/components/DataFiles/DataFilesModals/DataFilesPreviewModal.js

```diff
@@ -1,4 +1,4 @@
-import React, { useState } from 'react';
+import React from 'react';
```

Comment:
This modal is opened if user clicks on files ( `client/src/components/DataFiles/DataFilesListing/DataFilesListingCells.js`). I think we need to update that to say tapis-only files.   So I believe `api` should be checked there to make sure it is tapis